sql/core/src/test/scala/org/apache/spark/sql/execution/columnar/InMemoryColumnarQuerySuite.scala

```diff
@@ -485,16 +485,16 @@ class InMemoryColumnarQuerySuite extends QueryTest with SharedSQLContext {
     // Push predicate to the cached table.
     val df2 = df1.where("y = 3")
 
+    logWarning(s"ORIG QUERY PLAN:\n${df2.queryExecution.executedPlan}")
```

Comment:
Ur, shall we clean up these three `logWarning`s in this test suite? This will not be read.